build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -72,15 +72,30 @@ steps:
       node build/azure-pipelines/mixin
     displayName: Mix in quality
 
+  - script: |
+      set -e
+      yarn gulp install-sqltoolsservice
+    displayName: Install sqltoolsservice
+
+  - script: |
+      set -e
+      yarn gulp install-ssmsmin
+    displayName: Install ssmsmin
+
   - script: |
       set -e
       yarn gulp vscode-linux-x64
     displayName: Build
 
   - script: |
       set -e
-      yarn gulp install-sqltoolsservice
-    displayName: Install sqltoolsservice
+      yarn gulp compile-extensions
```

Comment:
We have to separately compile the extensions, since running the package task does not compile the extensions that we are looking for here.

An optimization could be made here to update this to only compile the extensions we want to compile, but its not that much more overhead just to compile them all.